python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/torch_ods_gen.py

```diff
@@ -226,6 +226,7 @@ def get_by_triple(self, key: Tuple[str, str, str]):
     "int": "Torch_IntType",
     "int[]": "TorchIntListType",
     "int?": "TorchOptionalIntType",
+    "int[]?": "TorchOptionalIntListType",
```

Comment:
There is some confusion in naming. "Tensor?[]": "AnyTorchOptionalTensorListType" means the tensor element is optional but "int[]?": "TorchOptionalIntListType" means the list is optional.

Can you help to change all list type to ListOfXXXType. Optional list type becomes OptionalListOfXXXType and when list element is optional ListOfOptionalXXXType.